db scripts/function_get_investigations_statistics.sql

```diff
@@ -121,5 +121,5 @@ BEGIN
 END;
 $BODY$;
 
-ALTER FUNCTION public.function_get_investigation_statistics(integer, integer[], date, date)
-    OWNER TO coronai;
\ No newline at end of file
+ALTER FUNCTION public.function_get_investigation_statistics(integer, integer[], timestamp, timestamp)
+    OWNER TO postgres;
```

Comment:
change back to coronai

Reply:
the OWNER

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1447/commits/dc79ef1ee7d06a477a2429a06360571043d7d062)